### PR TITLE
catchchallenger: new recipe

### PR DESCRIPTION
**Pull request by alphaonex86**

### CODE REVIEW

**Comment by korli** [comment]
The recipe needs to be named catchchallenger-2.2.0.0.recipe

File: games-rpg/catchchallenger/catchchallenger-2.2.0.0.txt [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	lib:libQt5Xml$secondaryArchSuffix
+	lib:libZstd$secondaryArchSuffix
```
Comment (korli) [inline games-rpg/catchchallenger/catchchallenger-2.2.0.0.txt]: lib:libzstd would be more correct.

File: games-rpg/catchchallenger/catchchallenger-2.2.0.0.txt [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libGL$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	devel:libQt5Xml$secondaryArchSuffix
+	devel:libZstd$secondaryArchSuffix
```
Comment (korli) [inline games-rpg/catchchallenger/catchchallenger-2.2.0.0.txt]: little z

**Comment by korli** [comment]
A few comment , otherwise a good job!

File: games-rpg/catchchallenger/catchchallenger-2.2.0.0.txt [hunk 3]
```diff
@@ -0,0 +1,62 @@
+SUMMARY="Game MMORPG/single player game. You can fight, farming, crafting, trading"
+DESCRIPTION="This game is a independent MMORPG or a single player game. You can fight, farming, crafting, trading, ..."
```
Comment (diversys) [inline games-rpg/catchchallenger/catchchallenger-2.2.0.0.txt]: Line should be wrapped at 80th character.

File: games-rpg/catchchallenger/catchchallenger-2.2.0.0.recipe [hunk 4]
```diff
@@ -1,5 +1,5 @@
 SUMMARY="Game MMORPG/single player game. You can fight, farming, crafting, trading"
-DESCRIPTION="This game is a independent MMORPG or a single player game. You can fight, farming, crafting, trading, ..."
+DESCRIPTION="This game is a independent MMORPG or a single player game."
```
Comment (Begasus) [inline games-rpg/catchchallenger/catchchallenger-2.2.0.0.recipe]: Suggestion:
```diff
-DESCRIPTION="This game is a independent MMORPG or a single player game."
+DESCRIPTION="This game is a independent MMORPG or a single player game. \
+You can fight, farming, crafting, trading, ..."
```

**Pull request merged**